src/main/kotlin/com/zopa/ktor/opentracing/OpenTracingServer.kt

```diff
@@ -17,26 +17,32 @@ import io.opentracing.propagation.TextMapAdapter
 import io.opentracing.tag.Tags
 import kotlinx.coroutines.asContextElement
 import kotlinx.coroutines.withContext
+import java.lang.Exception
 import java.util.Stack
 
 
-class OpenTracingServer(
-        val filters: List<(ApplicationCall) -> Boolean>
-) {
+class OpenTracingServer {
     class Configuration {
         val filters = mutableListOf<(ApplicationCall) -> Boolean>()
 
+        val tagsToAdd = mutableListOf<Pair<String, () -> String>>()
```

Comment:
i'm a bit confused that we have two tagsToAdd variables, can you please rename one of them